dasharo-compatibility/dcu.robot

```diff
@@ -147,7 +147,6 @@ DCU004.001 Verify SMMSTORE changes
 *** Keywords ***
 Prepare DCU Test Environment
     Run    rm -rf dcu
```

Comment:
Do we really want to remove it still, if it as added as a submodule, and not cloned later on?

Reply:
Applies to the other cases as well, if makes sense

Reply:
@EduKav1813 
I am working on reworking how DCU is used in OSFV and our PRs are in conflict :). I'll update the tests to use the submodule you added here: https://github.com/Dasharo/open-source-firmware-validation/pull/543 so you won't have to do it in this PR, ok?

Reply:
@philipandag Sure, in return I can help you test this change in your PR/reword

Reply:
It was quickly tested on a single test case here: https://github.com/Dasharo/open-source-firmware-validation/issues/541#issuecomment-2482804129

You can check the whole suite to make sure then